ergoCubSN000/hardware/inertials/waist-inertial.xml

```diff
@@ -7,4 +7,5 @@
     <param name="xsensmt_gyro_period">0.005</param>
     <param name="xsensmt_acc_period">0.005</param>
     <param name="xsensmt_mag_period">0.01</param>
+    <param name="sensor_name">waist_imu_0</param>
   </device>
```

Comment:
Done in https://github.com/robotology/robots-configuration/pull/614/commits/d2f159b82a5d9596ce7a597f65b40ee1ded05c7d (even if I typically prefer to avoid mixing cleanup and changes).